src/App.js

```diff
@@ -6,6 +6,7 @@ import ChatRoom from "./Components/ChatRoom";
 import Home from "./Components/Home";
 import NumberSumUp from "./Components/NumberSumUp";
```

Comment:
Hi Shashank, I think this portion of the code is not required.

Reply:
ok I will have a look and fix that